pegasus/forms/hoc_signup_2014.rb

```diff
@@ -28,16 +28,4 @@ def self.process(data)
       results.merge! location.to_solr if location
     end
   end
-
-  def self.solr_query(params)
```

Comment:
Those are both inline solr queries that don't use the forms' solr_query method. 

The first one is in a script that generates a csv. This is not accessible to clients. It also doesn't appear to be called from code anywhere.
The second is called on the server side to generate a list. Only organization_name_s and  location_city_s are exposed.